lib/licensee.rb

```diff
@@ -60,7 +60,7 @@ def confidence_threshold
     end
 
     def package_manager_files?
-      @package_manager_files ||= false
+      @package_manager_files || false
```

Comment:
Package manager files is set via `Licensee.package_manager_files = true`. If the instance variable is true, it returns the instance variable. If the instance variable is `false`, it'd fall through the `||=` (as I learned yesterday) and would be `false` either way.